src/elements/services/stylesheetService/IStylesheetService.ts

```diff
@@ -0,0 +1,10 @@
+import { TypedEvent } from "@node-projects/base-custom-webcomponent";
+import { IDesignItem } from "../../item/IDesignItem.js";
+import { IProperty } from "../propertiesService/IProperty.js";
+import * as csstree from 'css-tree';
+
+export interface IStylesheetService {
+    getDefiningRule(designItems: IDesignItem[], property: IProperty): csstree.RulePlain;
```

Comment:
we should not return types from external libs, cause we could switch to other implenentations
they should be a internal impl. detail